articles/deploying-machine-learning-models-using-flask/index.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/deploying-machine-learing-models-using-flask/
+title: Deploying Machine Learing models using Flask
+description: This tutorial will serve as an introduction on deploying Machine Learning models using Flask. We will go through various steps for building an end-to-end web application with inbuilt Machine Learning model using Flask.
+author: srishilesh-p-s
+date: 2020-12-00T00:00:00-00:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/deploying-machine-learing-models-using-flask/hero.jpg
```

Comment:
check all the URLs carefully. It says learing, but I think it should be learning, looking at the folder name.

Reply:
Thanks for that.